examples/angular/src/pages/ui/components/authenticator/reset-password/reset-password.component.html

```diff
@@ -1,4 +1,4 @@
-<amplify-authenticator initialState="forgotPassword">
+<amplify-authenticator initialState="resetPassword">
```

Comment:
This is incorrect, `resetPassword` is no longer an allowed value for `initialState`

Reply:
Oh weird if I run this example without the change it just takes me through a regular sign in flow and with the change it resets the password successfully